Automattic/newspack-newsletters
# Prevent duplication after insertion in Posts Inserted block

### All Submissions:






### Changes proposed in this Pull Request:



Closes #185

One caveat is that if the user _inserts_ posts, then _removes_ the inserted blocks, the post/s will still be unavailable in Post Inserter blocks (that is, treated as if inserted). But that's an edge case and I don't think it's that important. 

### How to test the changes in this Pull Request:

1. Reproduce #185 on master
2. Switch to this branch and try to reproduce again – the issue should not apprear

### Other information:
